### ISSUE TITLE
How to use BatchNormalization with tf.keras

### ISSUE THREAD

**Issue by JulietteV** [SciSharp/TensorFlow.NET#625]
Hello,
I'm trying to translate a Python Unet architecture into Tensorflow.NET using tf.keras but I don't find how to transpose this piece of Python code into C# :

```
def conv2d_block(input_tensor, n_filters, kernel_size=3, batchnorm=True):
    # first layer
    x = Conv2D(filters=n_filters, kernel_size=(kernel_size, kernel_size), kernel_initializer="he_normal",
               padding="same")(input_tensor)
    if batchnorm:
        x = BatchNormalization()(x)
    x = Activation("relu")(x)
    # second layer
    x = Conv2D(filters=n_filters, kernel_size=(kernel_size, kernel_size), kernel_initializer="he_normal",
               padding="same")(x)
    if batchnorm:
        x = BatchNormalization()(x)
    x = Activation("relu")(x)
    return x

```
In particular I don't understand how to use BatchNormalization in C#.

Can you help me ?

**Comment by openmll** [comment]
**Python version**
if batchnorm:
        x = BatchNormalization()(x)    

**TF.NET**
if (bn)
 {
                    conv = tf.layers.batch_normalization(conv, beta_initializer: tf.zeros_initializer,
                                                 gamma_initializer: tf.ones_initializer,
                                                 moving_mean_initializer: tf.zeros_initializer,
                                                 moving_variance_initializer: tf.ones_initializer, training: trainable);
}

actually, this code doesn't works solely. please refer the sample projects from **[SciSharp-Stack-Examples.](https://github.com/SciSharp/SciSharp-Stack-Examples)**

**Comment by Oceania2018** [comment]
`tf.layers.batch_normalization` has not been tested yet.

**Comment by Esther2013** [comment]
@JulietteV It's available in [TensorFlow.Keras v0.2.1](https://www.nuget.org/packages/TensorFlow.Keras/0.2.1)
